### DISCUSSION TITLE
HasMany with Attach

### DISCUSSION THREAD

**Discussion by fruitl00p** [laravel/nova-issues#3795] [Ideas & Feature Requests]
I know this feature request will probably be autoclosed but with the upcoming release of Nova v4 we're really looking forward to full compatibility / parity of the attaching of relations that have a HasMany property. (see also #520 and #3191 )

It basically boils down to the parity in having BelongsToMany (and thereby allowing attaching other resources) and the HasMany property that also should 'allow' attaching. 

And i can even imagine having the option to alter the button to "allow" creating new or attaching existing resources using 'allowCreate()' or 'disallowCreate()'

I look forward to some sort of official response ;)

**Comment by crynobone** [comment]
It's unlikely we can add this feature on the initial 4.0 release, sorry. But we always looking at [closed feature requests](https://github.com/laravel/nova-issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arequest+-label%3Adocumentation+-label%3Aimplemented) to see what's possible and can be maintained within the core.

**Comment by fruitl00p** [comment]
Thanks. Any tips on getting this (custom) fixed somehow?

**Comment by mrleblanc101** [comment]
I hope this can be added someday
